### PR TITLE
Validation: preserveView

### PR DESCRIPTION
**Pull request by ttorble**
User story [1907185](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1907185)

Guidance: https://review.docs.microsoft.com/en-us/help/contribute/versioning-for-contributors?branch=master

***
User story [1909902](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1909902)
(Same files)

### CODE REVIEW

**Comment by mikefrobbins** [comment]
We release new versions monthly. Please update the PR to include docs for version 7.2.0 and remove version 7.0.0. (Current versions).

This content is auto generated reference content. Please also update [the source in the azure-powershell repo](https://github.com/Azure/azure-powershell/tree/main/src/OperationalInsights/OperationalInsights/help). (Future versions)

File: azps-6.6.0/Az.OperationalInsights/Get-AzOperationalInsightsIntelligencePack.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ original_content_git_url: https://github.com/Azure/azure-powershell/blob/main/sr
 Gets the available Intelligence Packs.
 
 > [!NOTE]
-> Solutions is being deprecated, please use [az monitor log-analytics solution](https://docs.microsoft.com/en-us/cli/azure/monitor/log-analytics/solution?view=azure-cli-latest) and [Get-AzMonitorLogAnalyticsSolution](https://docs.microsoft.com/en-us/powershell/module/az.monitoringsolutions/get-azmonitorloganalyticssolution?view=azps-5.9.0) instead if this command.
+> Solutions is being deprecated, please use [az monitor log-analytics solution](/cli/azure/monitor/log-analytics/solution) and [Get-AzMonitorLogAnalyticsSolution](https://docs.microsoft.com/en-us/powershell/module/az.monitoringsolutions/get-azmonitorloganalyticssolution?view=azps-5.9.0) instead if this command.
```
[the remark this answers is not in the document]
Comment (mikefrobbins) [reply azps-6.6.0/Az.OperationalInsights/Get-AzOperationalInsightsIntelligencePack.md:17]: @v-andreaco This is autogenerated reference content. The note was probably added by the service team when the deprecation occurred. If any updates are being performed, please be sure to update the source which resides in the [azure-powershell repo](https://github.com/Azure/azure-powershell/tree/main/src/Monitor/Monitor/help).

We add and remove 5,000 auto-generated reference articles to this repo each month based on the versions in the source repo.

**Comment by v-andreaco** [comment]
> We release new versions monthly. Please update the PR to include docs for version 7.2.0 and remove version 7.0.0. (Current versions).
> 
> This content is auto generated reference content. Please also update [the source in the azure-powershell repo](https://github.com/Azure/azure-powershell/tree/main/src/OperationalInsights/OperationalInsights/help). (Future versions)

@mikefrobbins We're working from a list provided in the [User Story](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1907185). The docs in the source repo aren't currently listed in the spreadsheet attached to the US. @dbradish-microsoft Please advise.

Thanks!

**Comment by mikefrobbins** [comment]
> @mikefrobbins We're working from a list provided in the [User Story](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1907185). The docs in the source repo aren't currently listed in the spreadsheet attached to the US. @dbradish-microsoft Please advise.
> 
> Thanks!

@v-andreaco Any issue resolved in version 7.1.0 of the reference docs will resurface next month when 7.3.0 is released unless the source version is fixed. Also, all docs for version 7.1.0 will be deleted next month. This process occurs every month (adding a new version and removing an old one).

**Comment by dbradish-microsoft** [comment]
@mikefrobbins , lines 61 - 66.  The spreadsheet you looked at probably had a filter going in column I.  

@ttorble, autogenerated reference content has to be changed in the source.  Making updates to any folder except for \docs-conceptual\ in this repo could be over-written should the version be regenerated.  I'm really not concerned about autogenerated reference content as fully qualified URLs might be required for the autogen to build (?).  I'm not sure.  I'll update my SELECT statement to omit autogenerated content in the future.  Please close this PR and make a note in the work item that Azure PowerShell (any folder except for "docs-conceptual") and Azure CLI reference content (any folder with "docs-ref-autogen" in the name) was skipped.

**Pull request closed without merging**